### PR TITLE
Use NetworkManager always

### PR DESCRIPTION
**Pull request by lnussel**
Wicked lacks caretaking in Tumbleweed and is just slow to boot.
NetworkManager is good enough also for most common server use cases
in Tumbleweed so let's just use one technology for all roles.

YaST anyway allows to switch to wicked in the proposal screen. So those
who know that they need for advanced cases wicked can still have it.

### CODE REVIEW

File: package/skelcd-control-openSUSE.changes [hunk 1]
```diff
@@ -1,3 +1,9 @@
+-------------------------------------------------------------------
+Wed Jan 19 12:48:38 UTC 2022 - Ludwig Nussel <lnussel@suse.de>
+
+- Use NetworkManager always
```
Comment (dgdavid) [inline package/skelcd-control-openSUSE.changes:4]: If I'm not wrong, a reference to Bugzilla/JIRA/Github entry is needed to please OBS.

**Review by dgdavid — COMMENTED** [review]
In general it looks good to me, but I prefer having someone else reviewing it too before merging.

File: package/skelcd-control-openSUSE.changes (hunk 1 above)
Comment (lnussel) [reply package/skelcd-control-openSUSE.changes:4]: Not for Tumbleweed

**Review by imobachgs — APPROVED** [review]

File: package/skelcd-control-openSUSE.changes (hunk 1 above)
Comment (dgdavid) [reply package/skelcd-control-openSUSE.changes:4]: But is needed to please Jekins. See the result of the [public job](https://ci.opensuse.org/job/yast-skelcd-control-openSUSE-master/68/console)

> Stopping, missing new bugzilla or fate entry in the *.changes file.

**Comment by dgdavid** [comment]
Ok, we will need a second PR adding

* a Bugzilla / JIRA / Github entry
* fixing the version in the .spec file

**Review by mchf — APPROVED** [review]
If PM wishes so, then lgtm

File: package/skelcd-control-openSUSE.changes (hunk 1 above)
Comment (lnussel) [reply package/skelcd-control-openSUSE.changes:4]: looks like you really like to make your own lives harder :) filed follow up pr

File: package/skelcd-control-openSUSE.spec [hunk 2]
```diff
@@ -27,7 +27,7 @@
 #
 ######################################################################
 Name:           skelcd-control-openSUSE
-Version:        20211227
+Version:        20220121
```
[the remark this answers is not in the document]
Comment (lnussel) [reply package/skelcd-control-openSUSE.spec:30]: oops, sorry. no idea how that happened but maybe an indicator to finally find a way to not store store changes files in git anymore. OBS can generate them from git history if needed. filed pr 252

**Pull request merged**